paasta_tools/tron/client.py

```diff
@@ -109,3 +109,17 @@ def list_namespaces(self):
         """Gets the namespaces that are currently configured."""
         response = self._get('/api')
         return response.get('namespaces', [])
+
+    def get_job_content(self, job: str) -> dict:
+        return self._get(f"/api/jobs/{job}/")
+
+    def get_latest_job_run_id(self, job_content: dict) -> str:
+        job_runs = sorted(
+            job_content.get('runs', []),
+            key=lambda k: (k['end_time'] is None, k['end_time'], k['run_time']),
```

Comment:
I think it will make more sense to just return the last run by run number that is not scheduled.